documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -233,12 +238,26 @@ env:
 
 === Control Plane listener feature gate
 
-When enabled, the `ControlPlaneListener` feature gate configures Kafka brokers to use a separate listener for controller connections and for data connections (replication).
-This feature gate is currently in the alpha phase and is disabled by default.
-To enable it, use `+ControlPlaneListener` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
-This feature gate has to be disabled when upgrading from Strimzi 0.22 and earlier or when downgrading to Strimzi 0.22 and earlier.
+You can use the `ControlPlaneListener` feature gate to change the communication paths used for inter-broker communications.
+
```

Comment:
I wonder if it would be useful to introduce with a line on the concepts of _control_ and _data_planes first? And use those descriptions in the brackets.

Reply:
Yes, that makes sense. I attempted to summarise the control plane and data plane first. Please have a another look.

Reply:
Looks good